### PR TITLE
Final merge r1.6.0 main

### PR DESCRIPTION
**Pull request by ericharper**

### CODE REVIEW

File: tutorials/tts/Inference_ModelSelect.ipynb [hunk 1]
```diff
@@ -230,9 +231,6 @@
     "            \"mixerttsx\": \"tts_en_lj_hifigan_ft_mixerttsx\"\n",
     "        }\n",
     "        pretrained_model = spectrogram_generator2ft_hifigan.get(spectrogram_generator, \"tts_hifigan\")\n",
-    "    elif audio_generator == \"univnet\":\n",
```
Comment (Oktai15) [inline tutorials/tts/Inference_ModelSelect.ipynb:233]: @ericharper, could you return these three lines? Main has them.

**Review by Oktai15 — CHANGES_REQUESTED** [review]

File: tutorials/tts/Inference_ModelSelect.ipynb (hunk 1 above)
Comment (ericharper) [reply tutorials/tts/Inference_ModelSelect.ipynb]: Tried to revert the changes. Can you check if it's correct?

**Review by Oktai15 — APPROVED** [review]
Approve TTS changes.

**Review by titu1994 — APPROVED** [review]
Looks good. One minor comment is the GPT / BERT megatron `_enable_nvidia_optimizations()` is duplicated, maybe merge it to single function on main branch,

**Comment by ericharper** [comment]
> Looks good. One minor comment is the GPT / BERT megatron `_enable_nvidia_optimizations()` is duplicated, maybe merge it to single function on main branch,

Yes, we are in the process of consolidation megatron code across gpt/bert/t5

**Pull request merged**